### ISSUE TITLE
FEAT: Data Editor

### ISSUE THREAD

**Issue by jet10000** [beekeeper-studio/beekeeper-studio#808]
Please pay more attention to data sorting and editing work, for example, string fields can be displayed as textarea instead of just input. In addition, image upload function and so on. A similar tool is needed for data sorting, which can provide api for programming.

**Comment by rathboma** [comment]
Yes! We have plans to make editing easier by introducing a right-side sidebar, that way long strings can be in a text area, and JSON can be highlighted and formatted correctly for example.
